packages/remirror__extension-tables/src/table-extensions.ts

```diff
@@ -144,6 +149,43 @@ export class TableExtension extends NodeExtension<TableOptions> {
     return plugins;
   }
 
+  createPlugin(): CreateExtensionPlugin {
```

Comment:
Maybe we can directly use `createNodeViews` method here. The code will be simpler.

Reply:
We're using `updateExtensionPlugins` here (in the `onView` method),

This only checks the extension's `createPlugin` and `createExternalPlugins` methods.

[See here](https://github.com/remirror/remirror/blob/main/packages/remirror__core/src/builtins/plugins-extension.ts#L193-L216)